### DISCUSSION TITLE
Vercel don't react to deployed webapp

### DISCUSSION THREAD

**Discussion by ostermann-felix** [neuefische/questions#212] [Web] [answered]
Hello, this is my newest state of the medihelp app. In localhost everything works normal, but deployed the links don't work and nothng happens. [capstone-project-to8p.vercel.app](https://capstone-project-to8p.vercel.app/)

This is my repo for the app: https://github.com/ostermann-felix/capstone-project

![Bildschirmfoto 2022-10-05 um 11 53 35](https://user-images.githubusercontent.com/94837451/194033449-4216dc38-9f5c-4400-b694-b997d8abebb4.png)

After a while following is happening:
![Bildschirmfoto 2022-10-05 um 11 55 17](https://user-images.githubusercontent.com/94837451/194033711-69d293aa-1cba-4d5f-a369-0676a54db498.png)

I never saw this before and can't find anything simular online

**Comment by MerleMak** [comment, marked as answer]
Hi Felix, did you set your environment variable in Vercel for your MongoDB Connection string? :) This is a configuration in Vercel directly.

**Reply by ostermann-felix** [reply]
It stopped working within the last merge I did. Before that, everything was fine.

**Reply by ostermann-felix** [reply]
I run  now npm i -g vercel but it wants to add a second repo to my deployment before it sends the env variable...
